### ISSUE TITLE
This component does not work if the slot has any margin or padding properties.

### ISSUE THREAD

**Issue by niels-bosman** [maoberlehner/transition-to-height-auto-with-vue#14]
Not sure how to fix this but maybe something to note?

**Comment by WofWca** [comment]
With margins, this can help.
```diff
  .expand-enter-active,
  .expand-leave-active {
-   transition: height 1s ease-in-out;
+   transition: height 1s ease-in-out, margin 1s ease-in-out;
    overflow: hidden;
  }
  .expand-enter,
  .expand-leave-to {
    height: 0;
+   margin-top: 0 !important;
+   margin-bottom: 0 !important;
  }
```

**Comment by niels-bosman** [comment]
@maoberlehner Mind looking into this?

**Comment by maoberlehner** [comment]
Was slightly confused at first because this repository is only for demo purposes for the corresponding article. But just realized the npm repo that was created with my permission, links to this repo. This is a mistake. If this repository updates, nothing will happen to the npm package!

Please contact @whyayen
See https://github.com/maoberlehner/transition-to-height-auto-with-vue/issues/2#issuecomment-474809207

**Comment by maoberlehner** [comment]
It seems @whyayen has already merged your pull request: https://github.com/maoberlehner/transition-to-height-auto-with-vue/pull/15#issuecomment-634401136

@whyayen please change the repository in the npm package. Thanks!

**Comment by whyayen** [comment]
I remember that I have already merged #15 . And I publish it with version 0.1.0. I will double check tonight. Thanks!

**Comment by whyayen** [comment]
@nehlis The bug has already fixed by @WofWca  in my [fork](https://github.com/whyayen/transition-to-height-auto-with-vue) in v0.1.0. You can upgrade `vue-transition-expand` to v0.1.0 if you install it with npm or yarn. Welcome PR or issue if you get any problems. Thanks.
